Makefile

```diff
@@ -18,7 +18,7 @@ restart: stop start # Restart project
 log: project-log # Show project logs
 
 python-requirements:
-	make docker-run-python \
+	make docker-run-tools \
```

Comment:
Can we change that line to `make docker-run-tool CMD="pip install -r requirements.txt" DIR=./application`?